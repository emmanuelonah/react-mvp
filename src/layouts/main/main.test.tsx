import { render } from '@testing-library/react';

import { Main } from '..';

describe('<Main/>', () => {
  test('should render Component', () => {
    const { container } = render(<Main>Hello React Testing Library</Main>);
    expect(container.firstChild).toMatchSnapshot();
  });
});
