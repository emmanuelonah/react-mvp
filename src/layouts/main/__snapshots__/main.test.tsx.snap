// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> should render Component 1`] = `
<main
  data-testid="main"
  id="main"
>
  Hello React Testing Library
</main>
`;
